build.xml

```diff
@@ -27,7 +27,7 @@
         <!-- Install dependencies through Composer -->
         <exec executable="php" failonerror="true">
             <arg value="composer.phar"/>
-            <arg value="install"/>
+            <arg value="update"/>
```

Comment:
Is this in the init target? Why is it changing to update? Is that ok for the first time?